gammapy/datasets/flux_points.py

```diff
@@ -487,7 +487,6 @@ def plot_fit(
         ax_residuals=None,
         kwargs_spectrum=None,
         kwargs_residuals=None,
-        axis_name="energy",
```

Comment:
The argument was unused?

Reply:
It was unused and anyway it should be contained within the `kwargs_spectrum`